packages/core-mobile/app/services/notifications/news/subscribeForNews.ts

```diff
@@ -0,0 +1,43 @@
+import Logger from 'utils/Logger'
+import Config from 'react-native-config'
+import fetchWithAppCheck from 'utils/httpClient'
+import { ChannelId } from '../channels'
+import { newsEvents } from './events'
+
+export async function subscribeForNews({
+  deviceArn,
+  channelIds
+}: {
+  deviceArn: string
+  channelIds?: ChannelId[]
+}): Promise<{ message: 'ok' }> {
+  const events =
+    channelIds === undefined
+      ? []
```

Comment:
there is no point in subscribing an [] so I think we can just exit early or throw error